cmd/kueue/main.go

```diff
@@ -153,6 +156,26 @@ func main() {
 		SecureServing:  true,
 		FilterProvider: filters.WithAuthenticationAndAuthorization,
 	}
+
+	if cfg.InternalCertManagement == nil || !*cfg.InternalCertManagement.Enable {
+		metricsCertPath := "/etc/kueue/metrics-certs"
```

Comment:
```suggestion
		metricsCertPath := "/etc/kueue/metrics/certs"
```

As I mentioned in KEP, we might want to use additional subdirectory to make room for future configuration.
WDYT?

Reply:
Done.